tests/dragonfly/__init__.py

```diff
@@ -159,6 +159,13 @@ def stop_all(self):
     def __str__(self):
         return f"Factory({self.args})"
 
+    @property
+    def dfly_path(self):
```

Comment:
Maybe use `os.path.basedir` here? hard-coding the test binary name is a bit annoying if people try to bisect regressions etc